openc3/lib/openc3/packets/packet_item.rb

```diff
@@ -168,7 +166,6 @@ def states=(states)
         end
 
         @states = upcase_states
-        @state_colors ||= {}
```

Comment:
I want @state_colors to remain nil as set in the constructor unless they are actually populated